s/frontend/components/reviews/my-review/index.css.ts

```diff
@@ -8,6 +8,12 @@ export default css`
 	padding: 20px;
 }
 
+img {
+	width:50px;
```

Comment:
i'm noticing that these styles are similar to the other styles.

instead of styling the avatars the same way in many components,

let's make a new component called `<namaka-avatar>` which styles the avatars in a consistent way, and then we can use the consistent avatar component anywhere we want, and it always appears properly -- and the styles for avatars are then in one location, with the component